python/paddle/v2/framework/tests/test_assign_op.py

```diff
@@ -0,0 +1,21 @@
+import op_test
+import numpy
+import unittest
+
+
+class TestAccuracyOp(op_test.OpTest):
+    def setUp(self):
+        self.op_type = "assign"
+        x = numpy.random.random(size=(100, 10))
```

Comment:
Here, the input type is LoDTensor.  Whether needs to add the unit test to others types?

Reply:
Other types of variable are not exposed to Python well. It will be tested during if-else operator's unit tests. 

It will be added in the following PRs